Src/DLLEntry.pas

```diff
@@ -3,43 +3,30 @@
 interface
 
 uses
-  SysUtils, ToolsAPI, DFFWizard, System.Classes, Winapi.Windows, vcl.Graphics;
+  SysUtils, ToolsAPI, DFFWizard;
 
 function InitWizard(const BorlandIDEServices: IBorlandIDEServices;
   RegisterProc: TWizardRegisterProc; var Terminate: TWizardTerminateProc): Boolean; stdcall;
-
-
 exports
   InitWizard name WizardEntryPoint;
 
 implementation
 
 const
-  InvalidIndex = -1;
-
+  INVALID_INDEX = -1;
 
 var
-  FWizardIndex: Integer = InvalidIndex;
-
-procedure AddSplashScreenInfo;
-begin
-  if Assigned(SplashScreenServices) then
-  begin
-    var lIcon := LoadBitmap(HInstance, 'nogen_SplashScreen_icon');
-
-    SplashScreenServices.AddProductBitmap('nogen V1.0', lIcon);
-  end;
-end;
+  WizardIndex: Integer = INVALID_INDEX;
 
 procedure FinalizeWizard;
 var
   WizardServices: IOTAWizardServices;
 begin
-  if FWizardIndex <> InvalidIndex then
+  if WizardIndex <> INVALID_INDEX then

```

Comment:
Wouldn't the index be invalid if it was anything less then zero (`index<=0`) rather than just `-1`?